test/components/creeps/creepActions.test.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="../../../typings/globals/screeps/index.d.ts"/>
```

Comment:
Hmm, I can't seem to figure this out. I guess we could let it stay there for now.

Just put in `/* tslint:disable:no-reference */` on the beginning of all the files that has `/// <reference path="..." />` as to not make tslint angry.